src/main/scala/io/viash/platforms/nextflow/NextflowAuto.scala

```diff
@@ -17,9 +17,24 @@
 
 package io.viash.platforms.nextflow
 
+import scala.math.pow
+import scala.collection.immutable.ListMap
+
 case class NextflowAuto(
   simplifyInput: Boolean = true,
   simplifyOutput: Boolean = true,
   transcript: Boolean = false,
-  publish: Boolean = false
-)
\ No newline at end of file
+  publish: Boolean = false,
+  labels: ListMap[String, String] = ListMap(
+    NextflowAuto.labelIterator.take(9).map(s => (s"mem${s}gb", s"memory = ${s}.Gb")) ++
+    NextflowAuto.labelIterator.take(7).map(s => (s"cpu$s", s"cpus = $s")) : _*
+  )
+)
+
+object NextflowAuto {
+  // Returns 1, 2, 5, 10, 20, 50, 100 ...
+  // Stops after 9th power as Ints overflow at the 10th power (we'd actually still get 2 proper values)
+  def labelIterator: Seq[Int] = 
```

Comment:
I'm now wondering what would make more sense. 1gb, 2gb, 5gb, 10gb, 20gb, 50gb, 100gb, 200gb, 500gb, and then .. 1tb, 2tb, 5tb... ? 

Wdyt

Reply:
As discussed, this is now 1gb, 2gb, 4gb, 8gb, ..., 256gb, 512gb, 1tb, 2tb, ... upto 512tb
And cpus 1, 2, 5, 10, 20, ... upto 1000